src/ActionList/Item.tsx

```diff
@@ -0,0 +1,88 @@
+import type {IconProps} from '@primer/octicons-react'
+import React from 'react'
+import styled from 'styled-components'
+import {get} from '../constants'
+
+interface ItemPropsBase extends React.ComponentPropsWithoutRef<'div'> {
+  text: string
+  description?: string
+  descriptionVariant?: 'inline' | 'block'
+  leadingVisual?: React.FunctionComponent<IconProps>
+  leadingVisualSize?: 16 | 20
+  size?: 'small' | 'medium' | 'large'
+  variant?: 'default' | 'singleSelection' | 'multiSelection' | 'danger' | 'static'
+}
+interface ItemPropsWithRenderItem extends Partial<ItemPropsBase> {
+  renderItem: (props: ItemProps) => JSX.Element
+}
+export type ItemProps = ItemPropsBase | ItemPropsWithRenderItem
+
+const StyledItem = styled.div<{variant: ItemProps['variant']}>`
+    position: relative;
+    display: flex;
+    align-items: start;
+    border-radius: ${get('radii.2')};
+    font-weight: normal;
+    color: ${({variant}) => (variant === 'danger' ? get('colors.text.danger') : 'inherit')};
+    text-decoration: none;
+    border: 0;
+    background: none;
+    text-align: start;
+    margin: 0;
+    padding: calc((${get('space.3')} - ${get('space.1')}) / 2) ${get('space.2')};
+
+    @media (hover: hover) and (pointer: fine) {
```

Comment:
Why is this `@media` query needed?

Reply:
It’s used to avoid applying hover styles to mobile devices on tap.